src/index.hot.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { AppContainer as Hot } from 'react-hot-loader';
+
+import RootRender from '../src';
+
+const render = () => {
```

Comment:
No, I mean, in your original version, you need to re-require the Root component in accept handler, but in this version, that re-require is gone. That's what I mean.